src/main/java/frc/team4909/robot/sensors/Stream.java

```diff
@@ -1,103 +1,76 @@
-
-      MatOfPoint2f dst = new MatOfPoint2f(
-                new Point(0, 0),
-                new Point(height1-1,0),
-                new Point(0,length1-1),
-                new Point(height1-1,length1-1)      
-                );
-      Mat hatchWatchMatrix = Imgproc.getPerspectiveTransform(src, dst);
+      //Cameras 3 and 4 have mutual toggle function.
+      UsbCamera camera0 = CameraServer.getInstance().startAutomaticCapture(0); //Function: Drive
+      UsbCamera camera1 = CameraServer.getInstance().startAutomaticCapture(1); //Function: Line
+      UsbCamera camera2 = CameraServer.getInstance().startAutomaticCapture(2); //Function: Hatch 
+      UsbCamera camera3 = CameraServer.getInstance().startAutomaticCapture(3); //Function: Cargo
+      camera0.setResolution(160, 120);    /******************************************** */
+      camera1.setResolution(160, 120);    /**********************  Set   ************** */
+      camera0.setFPS(21);                 /********************  Camera  ************** */  
+      camera1.setFPS(16);                 /*************  Specifications    *********** */
+      camera2.setResolution(160, 120);    /******************************************** */
+      camera3.setResolution(160, 120);    /******************************************** */
+      camera2.setFPS(10);                 /******************************************** */
+      camera3.setFPS(10);                 /******************************************** */
+      //////VideoSink intakeServer = CameraServer.getInstance().getServer("Intake Cam");
+      CameraServer intakeServer = CameraServer.getInstance();//.addCamera(intakeServer);
+      ///////SmartDashboard.putData(intakeServer);
+      //CvSink cvSink1 = CameraServer.getInstance().getVideo();//CvSource outputStream = CameraServer.getInstance().putVideo("Source 1", 160, 120);//CvSink cvSink2 = CameraServer.getInstance().getVideo();//CvSource outputStream2 = CameraServer.getInstance().putVideo("Source 2", 160, 120);//CvSink cvSink3 = CameraServer.getInstance().getVideo();//CvSource outputStream3 = CameraServer.getInstance().putVideo("Source 3", 160, 120);//CvSink cvSink4 = CameraServer.getInstance().getVideo();//CvSource outputStream4 = CameraServer.getInstance().putVideo("Source 4", 160, 120);
+      //Mat source1 = new Mat();//Mat source2 = new Mat();//Mat output1 = new Mat();//Mat output2 = new Mat();//Mat source3 = new Mat();//Mat source4 = new Mat();//Mat output3 = new Mat();//Mat output4 = new Mat();
+      /*Mat image1 = new Mat(cvSink1.grabFrame(source1));int height1 = image1.rows();int length1 = image1.cols();MatOfPoint2f src = new MatOfPoint2f(new Point((1/6)*height1, 0),new Point((5/6)*height1,0),new Point(height1-1,length1-1),new Point(0,length1-1));
+      MatOfPoint2f dst = new MatOfPoint2f(new Point(0, 0),new Point(height1-1,0),new Point(0,length1-1),new Point(height1-1,length1-1));Mat hatchWatchMatrix = Imgproc.getPerspectiveTransform(src, dst);*/
```

Comment:
It looks like none of this matrix stuff is being used. How about just deleting all of it?